src/libsync/configfile.cpp

```diff
@@ -1113,7 +1113,10 @@ void ConfigFile::setAutomaticLogDir(bool enabled)
 
 QString ConfigFile::logDir() const
 {
-    const auto defaultLogDir = QString(configPath() + QStringLiteral("/logs"));
+    const auto defaultLogDir = QString(
+        QStandardPaths::writableLocation(QStandardPaths::AppDataLocation)
+        + QStringLiteral("/logs")
+    );
```

Comment:
that would create some behavior changes ?

Reply:
Yes, because I find it better to place the log files into the data location (`$XDG_DATA_DIR`) instead of the config location like the [logpath of the sync run logs](https://github.com/nextcloud/desktop/pull/7665/files#diff-6d359e2f56dc9b47e9108fd1467f60f3a05ba084b88d028218b054d71b7dc03dL35).

And then I use this logDir also for the sync run logs.